lib/vdsm/storage/outOfProcess.py

```diff
@@ -30,6 +30,7 @@
 import weakref
 
 from functools import partial
```

Comment:
We separate standard library moudules and 3rd party modules with blank line. You change remove the line separating "import six" and "from functools import partial".